controllers/control/k8ssandratask_controller.go

```diff
@@ -0,0 +1,212 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package control
+
+import (
+	"context"
+	"fmt"
+	cassapi "github.com/k8ssandra/cass-operator/apis/control/v1alpha1"
+	k8sapi "github.com/k8ssandra/k8ssandra-operator/apis/k8ssandra/v1alpha1"
```

Comment:
I don't like this naming, sorry. "k8s" is a common name for "Kubernetes", not "K8ssandra". As a reader, later on I would assume k8sapi to be "k8s.io/" apis.